code/game/objects/objs.dm

```diff
@@ -46,6 +50,48 @@ var/global/list/reagents_to_log = list(FUEL, PLASMA, PACID, SACID, AMUTATIONTOXI
 
 /obj/item/proc/is_used_on(obj/O, mob/user)
 
+/obj/proc/install_pai(obj/item/device/paicard/P as obj)
```

Comment:
When using a verb it IS relevant because then BYOND knows what type the variable is.

Otherwise no.